src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import Button from "../../inline/Button";
+
+import s from "./ArticleSpotlightCard.module.css";
+
+const TextCard = ({ eyebrowText, title, description, button }) => (
+  <div className={s.textCard}>
+    <div className={s.eyebrow}>{eyebrowText}</div>
+    <div className={s.title}>{title}</div>
+    <div className={s.description}>{description}</div>
+    <div className={s.actionRow}>
+      <Button
+        text={button.text}
+        internalLink={button.internalLink}
+        externalLink={button.externalLink}
+        onClick={button.onClick}
+      />
+    </div>
+  </div>
+);
+
+TextCard.propTypes = {
+  eyebrowText: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  button: PropTypes.shape(Button.propTypes).isRequired,
+};
+
+const ArticleSpotlightCard = ({
+  eyebrowText,
+  title,
+  description,
+  button,
+  backgroundImage,
+}) => (
+  <div className={s.bleedWrapper}>
+    <div
+      className={s.bleedBackground}
+      style={{
+        background: `center / cover no-repeat url("${backgroundImage}")`,
+      }}
+    />
+    <div className={s.bleedMainContent}>
+      <div className={s.gridParent}>
+        <div className={s.gridAreaCard}>
+          <div className={s.textCardWrapper}>
+            <TextCard
+              eyebrowText={eyebrowText}
+              title={title}
+              description={description}
+              button={button}
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+);
+
+ArticleSpotlightCard.propTypes = {
+  eyebrowText: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  button: PropTypes.shape(Button.propTypes).isRequired,
```

Comment:
I see that shape is used instead of arrayOf. Should arrayOf be used to give a developer the idea that they can use multiple buttons if that developer wanted to reuse this component??

Reply:
I didn't use `arrayOf()` because I don't know if that actually makes sense for this component. It's called the "Article Spotlight Card," and presumably it's meant to highlight one specific call to action. Until we actually have an example of having two buttons here, I don't think it makes sense to add another one, since it may allow a developer to abuse the component in a way that breaks the design aesthetic.